src/components/Admin/AIAnalyticsSummary.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Button, Card, Stack, Badge,
+} from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { AutoFixHigh, Groups } from '@edx/paragon/icons';
+import useAIAnalyticsSummary from '../AIAnalyticsSummary/data/hooks';
+
+const AnalyticsDetailCard = ({ onClose, data }) => (
+  <Card className="mt-3 mb-4">
+    <Card.Section>
+      <Badge variant="light" className="mb-3 font-weight-semibold">
+        Beta
+      </Badge>
+      <Stack gap={1} direction="horizontal">
+        <p className="card-text text-justify small">{data || 'No insights found..'}</p>
```

Comment:
Questions:

1. `'No insights found..'` message will render if we received the data from Analytics service but no data received from LMS? 
2. Are two dots intentional at the end of message?

Reply:
@muhammad-ammar 
1. Yes, correct. Specifically if `learner_progress`/`learner_engagement` is null or empty.
2. Yes, they were intentional.